src/core/ddsc/include/dds/ddsc/dds_public_status.h

```diff
@@ -248,7 +248,7 @@ dds_get_inconsistent_topic_status (
  */
 DDS_EXPORT dds_return_t
 dds_get_publication_matched_status (
-  dds_entity_t writer,
```

Comment:
This really only takes a writer, so I think "writer" is the better choice.

Reply:
Applied and fixed.